service/dap/server.go

```diff
@@ -209,6 +235,8 @@ func NewServer(config *service.Config) *Server {
 		variableHandles:   newVariablesHandlesMap(),
 		args:              defaultArgs,
 		exceptionErr:      nil,
+		inReverseMode:     inReverseMode,
```

Comment:
This flag can just be replaced by checking `s.listener == nil`. You can either make a helper method or check it directly.